ballerina/tests/csv_io.bal

```diff
@@ -525,6 +532,33 @@ isolated function testTableContent() {
     }
 }
 
+@test:Config {}
+isolated function testTableContent2() {
+    string filePath = RESOURCES_BASE_PATH + "datafiles/io/records/sample5.csv";
+    float expectedValue = 60001.00;
+    float total = 0.0;
+
+    var csvChannel = openReadableCsvFile(filePath);
```

Comment:
Shall we update the method signature with `returns error?` and use `check` here ? Then we could remove unwanted if-else conditions. Check for other places as well.

Reply:
Fixed in 54974f8